src/styles/colors.tsx

```diff
@@ -2,16 +2,19 @@
 // from https://www.figma.com/design/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System?node-id=2100-4881&node-type=frame&t=vKGGXrs3Torz7kFE-0
 enum Colors {
   // backgrounds
+  background = '#FFFFFF',
   backgroundInverse = '#2E3338',
 
   // text
   textPrimary = '#2E3338',
+  textInverse = '#FFFFFF',
 
   // borders, shadows, highlights
   shadow = '#2E3338',
   softShadow = 'rgba(156, 164, 169, 0.4)',
   lightShadow = 'rgba(48, 46, 37, 0.15)',
   barShadow = 'rgba(129, 134, 139, 0.5)',
+  loadingSkeletonHighlight = '#FFFFFF',
```

Comment:
supernit: wondering if we should use `skeletonPlaceholderHighlight`?
Unless maybe we wanna add more things prefixed with `loading`?

Reply:
yes sorry it's not clear here but i will also create a color for the loading indicator, although the spinner doesn't have to be directly related to the skeleton loader. i can rename it!